src/tools/jbuild

```diff
@@ -17,6 +17,8 @@
    (flags (:standard (:include ../ocaml-flags-standard.sexp) (:include ../ocaml-context-flags.sexp)))
    (libraries (opam-client))))
 
+(include opam-putenv.inc)
```

Comment:
Not really an issue, but we should be aware that we lose on Windows the property that opam can be used from a single installed binary (well, given that all its dependencies are there on course)
In any case, for Windows, we'll need some kind of installer, so that property isn't as valuable that it is on Linux. And it's in any case not a requirement.

Reply:
It's not completely lost - it's just that the user would otherwise have to ensure that they are using the correct `opam.exe`. You really don't want to see the gymnastics required to eliminate `opam-putenv.exe` completely :wink: